holidays/countries/nepal.py

```
@@ -0,0 +1,441 @@
+        2020: (NOV, 16),
+        2022: (OCT, 26),
+        2023: (NOV, 14),
+    }
+
+    # Chhath Parva.
+    CHHATH_PUJA_DATES = {
+        2025: (OCT, 27),
+    }
+
+
+class NP(Nepal):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)